mlflow/genai/judges/prompts/correctness.py

```diff
@@ -30,6 +32,15 @@
 
 If the claim is fully supported by the document in the context of the question, you must say "The response is correct" in the rationale. If the claim is not fully supported by the document in the context of the question, you must say "The response is not correct"."""  # noqa: E501
 
+CORRECTNESS_TRACE_FALLBACK = f"""{CORRECTNESS_BASE_INSTRUCTIONS}
+
+Extract the question, expected response/facts, and actual response from the following trace and \
```

Comment:
@BenWilson2 We're using inconsistent terminology. Can we make the terminology consistent with the `CORRECTNESS_BASE_INSTRUCTIONS` that we're appending to?

The base instructions refer to `question`, `claim`, and `document`, but this fallback refers to `question`, `expected response/facts`, and `actual response`.